kibana/templates/_helpers.tpl

```diff
@@ -41,3 +41,16 @@ heritage: {{ .Release.Service }}
 {{ toYaml .Values.labels }}
 {{- end }}
 {{- end -}}
+
+{{/*
+Use the fullname if the serviceAccount value is not set
+*/}}
+{{- define "kibana.serviceAccount" -}}
+{{- if and .Values.rbac.serviceAccountName ( not ( eq .Values.podSecurityPolicy.name "" ) ) -}}
```

Comment:
Why are you checking `podSecurityPolicy.name` value if you're not using `kibana.serviceAccount` in `kibana/templates/podsecuritypolicy.yaml`?

Reply:
I would prefer removing `kibana.serviceAccount` template and keeping `ClusterRole`, `ClusterRoleBinding` and `ServiceAccount` name consistent with what we have in most other charts (you can check `Logstash` chart for that).